### ISSUE TITLE
UWPX hangs after receiving contact request

### ISSUE THREAD

**Issue by bluszcz** [UWPX/UWPX-Client#107]
## Bevor you create a new issue:

```
- [x] I searched for similar issues and did not find one
- [x] I'm using the latest version available in the [Windows Store](https://www.microsoft.com/store/apps/9NW16X9JB5WV)
```

## I'm submitting a...:

 Bug report (I searched for similar issues and did not find one) 
 

## Current behavior:

I am receiving contact request which opens window with two buttons, Accept and Reject. After clicking any of the buttons application hangs. It does not send the response.

## Expected behavior:

Accept or reject the requesting without hanging the app.

## Minimal reproduction of the problem with instructions:

1. Open the app
2. Open other application with other xmpp account
3. Ask for subscription

## Environment:

```
App Version(s): 
v.X.Y.Z <!-- Which version of the App are you using (e.g. v.0.2.0) -->

Windows 10 Version Number: <!-- https://en.wikipedia.org/wiki/Windows_10_version_history -->
- [x] 1903
- [ ] 1809
- [ ] 1803
- [ ] 1709
- [ ] 1703
- [ ] 1607
- [ ] 1511
- [ ] 1507
- [ ] Insider Build (build number: )
- [ ] Misc:

Device form factor:
- [x] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub

Where did you got the APP from?
- [x] [Windows Store](https://www.microsoft.com/store/apps/9NW16X9JB5WV)
- [ ] Self-build, using a provided release source
- [ ] Self-build, repo cloned at [dd.mm.yyy] <!-- When did you clone the repo! -->
- [ ] Misc, got it from... <!-- Please tell us your source! -->

```

**Comment by COM8** [comment]
Thanks for reporting!

Does this happen all the time or only randomly?

**Comment by bluszcz** [comment]
All the time, unfortunately. It happens only on one pc at home - on 2 others it works well.

**Comment by COM8** [comment]
Ok thanks. I will look into it.  
I think I know the reason for this one 😉

**Comment by bluszcz** [comment]
Sweet, do you need any more info by any chance?

**Comment by COM8** [comment]
One thing: which server provider do you use and which server provider does your contact use?

**Comment by bluszcz** [comment]
jabberpl.org

**Comment by COM8** [comment]
I'm having trouble reproducing this reliably.
I noticed this behavior also, but until now I'm not able to reproduce it.
I tried registering accounts at jabberpl.org, bit I'm not getting it preproduced.
What other client do you use to send subscription requests?

**Comment by bluszcz** [comment]
Dino or and psi-plus

**Comment by COM8** [comment]
Should be fixed with the DB rework #146.
Feel free to reopen in case this is still an issue.
